convert2rhel/subscription.py

```diff
@@ -1097,7 +1097,7 @@ def update_rhsm_custom_facts():
     the conversion with the candlepin server, thus, propagating the
     "breadcrumbs" from convert2rhel as RHSM facts.
     """
-    if not tool_opts.no_rhsm:
+    if not tool_opts.no_rhsm and not "CONVERT2RHEL_DISABLE_TELEMETRY" in os.environ:
```

Comment:
You can move this extra condition to be on main, like the patch I posted on the comment of the PR.

Reply:
@r0x0d What would be advantage of having the condition in main? Because I've been wondering about it too and then I realized, that having it in the function with message would be more systematic and safer for usage in future (if it would be used but I feel it won't be :D) Is there any risk? Or just keep the condition simpler?

Reply:
Thanks for the clarification! I did all other changes and updated the PR. I was waiting to your response for this (which I see after the update).

I would rather keep the condition in the method, I like that costumer can see, that the telemetry was really skipped and having it just once in the code.

Reply:
Cool! That's fine by me. Thanks for popping up the question.